cmd/deploy_delete.go

```diff
@@ -45,6 +52,6 @@ func newDeleteDeploymentCmd(deployConfig *deployCommandConfig) *cobra.Command {
 		},
 	}
 
-	deleteDeploymentCmd.PersistentFlags().StringVarP(&deployConfigFile, "file", "f", "app-deploy.yaml", "The file name to use for the deployment configuration.")
+	deleteDeploymentCmd.PersistentFlags().StringVarP(&deployConfigFile, "file", "f", "app-deploy.yaml", "Name of the deployment configuration you want to use.")
```

Comment:
I liked having "file" in here because it must be a valid filename. So maybe we use
`Filename of the deployment configuration you want to use`